README.md

```diff
@@ -1,2 +1,19 @@
 # code-review-bot
-code-review-bot
```

Comment:
The installation instructions assume the user has Homebrew installed. Consider adding a check or instructions for users who may not have Homebrew.